src/global/utils/auth.tsx

```diff
@@ -54,18 +55,24 @@ export const storeToken = (egoToken: string) => {
 	Cookies.set(EGO_JWT_KEY, egoToken);
 };
 
-export const logOut = () => {
-	Cookies.remove(EGO_JWT_KEY);
-};
-
 export function AuthProvider({ children }: { children: ReactNode }) {
 	const storedToken = getStoredToken();
 	const [egoJwt, setEgoJwt] = useState(storedToken || '');
+	const router = useRouter();
 	const path = usePathname();
-	const initLoginState = path === '/logging-in' && !egoJwt.length ? true : false;
-	const [loggingIn, setLoggingIn] = useState(initLoginState);
+	const loginStateOnPageLoad = path === '/logging-in' && !egoJwt.length;
```

Comment:
https://github.com/icgc-argo/rdpc-ui/pull/39/files#diff-09e9a1c3d8949b041f6220ac58c961358436d20050a933e1f9bf52a9cd93428cR38

is this line doing the same logic? Is it covered by this line in AuthProvider which covers if a user directly goes to `logging-in` route for some reason

Reply:
Ah so I added this to `/logging-in` to prevent a User navigating to /logging-in and then clicking 'Log In' while the request completes. I couldn't get Next server/client to play nicely together here and Header frequently would load Login button on page load, then update to render loading animation. 

I will take another look to see if this can be streamlined but it's definitely something I had to go back-and-forth over several times.